pants.toml

```diff
@@ -13,6 +13,7 @@ backend_packages = [
 
   # Also enable Java to demonstrate cross-language support.
   "pants.backend.experimental.java",
+  "pants.backend.shell",
```

Comment:
Similar to https://github.com/pantsbuild/example-jvm/pull/28#discussion_r1316664344, this could be left off, to keep the example simpler.

Reply:
Yes, thanks, fixed!